### ISSUE TITLE
Issue with MCC metric computation

### ISSUE THREAD

**Issue by adallak** [h2oai/h2o-3#16329]
I am reading the documentation for the MCC metric available at https://docs.h2o.ai/h2o/latest-stable/h2o-py/docs/metrics.html and 
it says that "The absolute MCC (a value between 0 and 1, 0 being totally dissimilar, 1 being identical)." However, MCC is a value between -1 and 1. The value MCC = 0 means that the prediction behaves like a random guess, and MCC = -1 means that the algorithm predicts labels completely wrong. I tested and it seems H2O always reports the absolute value of MCC, which in my opinion is incorrect.  Please verify that the reported MCC is correct.
Thank you.

**Comment by wendycwong** [comment]
@adallak :  You are correct.  Here is a link to the definition: https://www.voxco.com/blog/matthewss-correlation-coefficient-definition-formula-and-advantages/

**Comment by maurever** [comment]
Hi @adallak. Our documentation shows a mismatch between MCC and Absolute MCC metrics. We need to clarify the API and the doc as well. Thanks for catching this!
